ui/src/api/mixins/ApiResource.ts

```diff
@@ -32,6 +32,10 @@ export function apiResourceMixin (rootUrl: string, separator: string): Mixin {
       get actions (): Actions {
         return commonActions(this, this._additionalActions)
       }
+
+      get inlineRepresentation () {
```

Comment:
Not sure I understand this

Reply:
Right, this is a little annoying technicality, for which I could not find a good solution. 

Take [Table#columnMappings](https://github.com/zazuko/cube-creator/pull/169/files#diff-952eebae7242509177c8aa88b60ec294a4c0fd9f04e5a20554993a7f2464e184R51) for an example. It is typed as an array of `Link<ColumnMapping>`. A link is deliberately typed as a plain resource with a `load` method to prevent accidentally adding triples about the linked resource inside the Table's graph.

But reconsidering this, an explicit cast can work just as well without adding meaningless properties

Reply:
TIL what a `Link` is :)